src/openms/source/FORMAT/MzTab.cpp

```diff
@@ -1412,78 +1448,85 @@ namespace OpenMS
   {
   }
 
-  void MzTab::addPepEvidenceToRows(const vector<PeptideEvidence>& peptide_evidences, MzTabPSMSectionRow& row, MzTabPSMSectionRows& rows)
+
+  void MzTab::addPepEvidenceToRows(const vector<PeptideEvidence>& peptide_evidences, MzTabPSMSectionRow& row)
   {
-    if (!peptide_evidences.empty())
+    if (peptide_evidences.empty())
     {
-      for (Size i = 0; i != peptide_evidences.size(); ++i)
-      {
-        // get AABefore and AAAfter as well as start and end for all pep evidences
-
-        // pre/post
-        // from spec: Amino acid preceding the peptide (coming from the PSM) in the protein
-        // sequence. If unknown “null” MUST be used, if the peptide is N-terminal “-“
-        // MUST be used.
-        if (peptide_evidences[i].getAABefore() == PeptideEvidence::UNKNOWN_AA)
-        {
-          row.pre = MzTabString("null");
-        }
-        else if (peptide_evidences[i].getAABefore() == PeptideEvidence::N_TERMINAL_AA)
-        {
-          row.pre = MzTabString("-");
-        }
-        else
-        {
-          row.pre = MzTabString(String(peptide_evidences[i].getAABefore()));
-        }
+      // report without pep evidence information
+      row.pre = MzTabString("null");
```

Comment:
doesn't an empty MzTabString default to "null" when written?
Saves a string comparison when initialized.